flexmeasures/api/v3_0/sensors.py

```diff
@@ -582,3 +582,59 @@ def post(self, sensor_data: dict):
         db.session.commit()
         sensor.resolution = sensor.event_resolution
         return sensor_schema.dump(sensor), 201
+
+
+
+    @route("/<id>", methods=["PATCH"])
+    @use_args(partial_sensor_schema)
+    @use_kwargs({"db_sensor": SensorIdField(data_key="id")}, location="path")
+    @permission_required_for_context("update", ctx_arg_name="db_sensor")
+    @as_json
+    def patch(self, sensor_data: dict, id: int, db_sensor: Sensor):
+        """Update an sensor given its identifier.
+
+        .. :quickref: Sensor; Update a sensor
+
+        This endpoint sets data for an existing sensor.
+        Any subset of sensor fields can be sent.
+
+        The following fields are not allowed to be updated:
```

Comment:
Where is this taken care of? That might require `dump_only=True` on the schema field for `generic_asset_id`.
And maybe adding `id = ma.auto_field(dump_only=True)`.